AutomationUtils/Utils/TestsUtils.cs

```diff
@@ -141,5 +147,16 @@ private static int SkipLineBreaksAndCommentsInExamplesTable(List<string> fileLin
 
             return iterator;
         }
+
+        public static Dictionary<string, List<string>> AllFeatureFilesAndTheirContent()

```

Comment:
Can we use property to init it only once and do not run full method each time we call it?